Classes/Controller/SeminarController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace OliverKlee\Seminars\Controller;
+
+/**
+ * This file is part of the "seminars" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Utility\DebuggerUtility;
+
+/**
+ * Controller of news records
+ *
+ */
+class SeminarController extends \TYPO3\CMS\Extbase\Mvc\Controller\ActionController
+{
+
+    /**
+     * @var \Tx_Seminars_Mapper_Event
+     */
+    protected $eventRepository;
```

Comment:
Please let's call it `$eventMapper` (and rename the inject method accordingly. And also please add the `null` default value (to communicate that we have didn't forget it).